validator/webui/index.html

```diff
@@ -252,7 +171,9 @@
         selectAndFocusError: function(index) {
           this.$.listbox.selected = index;
           this.$.listboxItems.render();
-          this.$.listbox.selectedItem.scrollIntoView();
+          if (this.$.listbox.selectedItem) {
+            this.$.listbox.selectedItem.scrollIntoView();
+          }
```

Comment:
Moved to outside method to check if errno is in range with validationresult.errors.